dotlottie-rs/src/lottie_renderer/mod.rs

```diff
@@ -194,14 +194,14 @@ impl LottieRenderer {
             ));
         }
 
+        self.thorvg_canvas.sync();
```

Comment:
to fix linting issue in the CI
```suggestion
        let _ = self.thorvg_canvas.sync();
```

Reply:
Oops, I missed that. Updated